0.11/Dockerfile

```diff
@@ -31,7 +31,11 @@ RUN buildDeps=" \
 	&& make install \
 	&& cd / \
 	&& rm -rf /usr/src/thrift \
-	&& curl -k -sSL "https://storage.googleapis.com/golang/go1.4.linux-amd64.tar.gz" -o go.tar.gz \
+	&& if [ $(uname -m) = "aarch64" ]; then \
```

Comment:
I would recommend you manage your own Dockerfile and the go runtime you need on top of the thrift compiler Dockerfile given here.  You are likely to have specific go version requirements different from others, and this project would not be able to maintain all the variants.

Reply:
Note in the 0.12 version Dockerfile, the go runtime is not installed at all.